src/Magnum/Trade/AbstractImporter.h

```diff
@@ -821,6 +823,9 @@ class MAGNUM_EXPORT AbstractImporter: public PluginManager::AbstractManagingPlug
 
         /** @brief Implementation for @ref importerState() */
         virtual const void* doImporterState() const;
+
+        /** @brief Implementation for @ref openState() */
+        virtual void doOpenState(const void* state);
```

Comment:
Third and last comment: I would *also* put this next to the `doOpenData()`/`doOpenFile()` functions.